src/views/Books/BookDetail.jsx

```diff
@@ -1,11 +1,12 @@
+import { useParams } from 'react-router-dom'
 import { useEffect, useState } from 'react'
 import Book from '../../components/book/Book'
 import { getBookById } from '../../services/books'
 
 function BookDetail() {
-  const id = 1 // TODO: Use id from route
+  const { id } = useParams() // TODO: Use id from route
   const [book, setBook] = useState(null)
-
+  console.log('here', book)
```

Comment:
You can clean up the extra console logs just to make good habits (not needed, but cleans up the code quite a bit)